mysql_fdw.c

```diff
@@ -942,11 +950,15 @@ mysqlGetForeignPlan(PlannerInfo *root,RelOptInfo *baserel, Oid foreigntableid, F
 	 * field of the finished plan node; we can't keep them in private state
 	 * because then they wouldn't be subject to later planner processing.
 	 */
-	return make_foreignscan(tlist,
-							local_exprs,
-							scan_relid,
-							params_list,
-							fdw_private);
+	return make_foreignscan(tlist
```

Comment:
I switched the formatting around to allow for capturing the change (within the `#if`) as locally as possible while maintaining consistent formatting across the items in the argument list of `make_foreignscan`.

I'm also fine with only using a start of line comma on the line within the `#if` (maintaining previous formatting/trailing commas elsewhere); or encapsulating the whole `return` in a `#if`/`#else` macro block (possibly cleaner looking, but more duplicated lines).